src/Controls/src/Core/Compatibility/Handlers/ListView/Windows/CellControl.cs

```diff
@@ -450,8 +448,7 @@ void SetupContextMenu()
 				return;
 			}
 
-			CellContent.PointerReleased += OnClick;
-			CellContent.Holding += OnLongTap;
+			CellContent.RightTapped += OnCellContentRightTapped;
```

Comment:
See this comment about why I changed to use this event: https://github.com/dotnet/maui/issues/6490#issuecomment-1180857070

TL;DR: The docs say it's the best event to use for this, and it also works better than the previous two events combined.